programmers/난이도별/level02.프린터/Jaewon0702.py

```diff
@@ -0,0 +1,25 @@
+def solution(priorities, location):
+    original=0
+    while(priorities!=[]) :
+        m=max(priorities)
+        p=priorities.pop(0)
+        if p<m :
+            priorities.append(p)
+            if location>0 :
+                location-=1
+            else :
+                location=len(priorities)-1
+        else:     #이 이후로는 https://somjang.tistory.com/155를 참고하였다.
+            original+=1
+            if location == 0 :
+                break
+            else :
+                location-=1

```

Comment:
```suggestion
        if p<m :
            priorities.append(p)
            location = location - 1 if location > 0 else len(priorities)-1
            continue
        original+=1
        if location != 0:
            location -= 1
```
if의 깊이는 최대한 줄일수록 좋습니다!